.github/workflows/build+test.yml

```diff
@@ -19,8 +19,8 @@ jobs:
         swift-version: "${{ env.swift_version }}"
     - name: Check out sources
       uses: actions/checkout@v4
-    - name: Check formatting
-      run: swift package plugin --allow-writing-to-package-directory swiftformat --lint
+    #- name: Check formatting
+    #  run: swift package plugin --allow-writing-to-package-directory swiftformat --lint
```

Comment:
Temporarily disabling this because it builds the project which it cannot now that the SDK depends on the Rust bindings which are not checked in. The job has to build these before this can be uncommented. As the tool `cargo-swift` currently used for this is rather flaky in its ability to compile, this requires replacing this dependency with a script that does the same thing.